### PR TITLE
[Support Links] Change old email-to-case links to emails to SBL Help

### PR DESCRIPTION
**Pull request by billhimmelsbach**
Currently, there are some links to the old email-to-case forms that would be better served by straight emails to SBL Help.

## Changes

- remove links to old email-to-case form and instead link via email with relevant subjects

## How to test this PR

1. Do the old support links now guide to emails with the link text "email our support staff"?

## Screenshots

Screenshot 2024-03-28 at 9 50 12 AM
Screenshot 2024-03-28 at 9 51 44 AM

### CODE REVIEW

File: src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx [hunk 1]
```diff
@@ -25,8 +24,11 @@ export function FinancialInstitutionDetails({
       <Heading type='2'>{heading}</Heading>
       <Paragraph>
         To make a change to the email domains for your financial institution,{' '}
-        <Link href={sblHelpLink}>contact our support staff</Link>. To make a
-        change to any other data in this section, visit <Links.GLIEF />.
+        <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] View your financial institution profile: updating my financial institution information'>
```
Comment (meissadia) [inline src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx:27]: This component also gets rendered in the `Update institution` form, so a more generic `Subject` may be more fitting. 
```suggestion
        <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Financial institution profile: updating my financial institution information'>
```

**Review by meissadia — APPROVED** [review]
Looks great!

File: src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx (hunk 1 above)
Comment (billhimmelsbach) [reply src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx]: Great catch @meissadia! 🎉

[Fixed](https://github.com/cfpb/sbl-frontend/pull/330/commits/97df326c0bd2a38fdde0c3781b2ed5dbe6de9e5a)!

File: src/pages/Filing/FilingHome.tsx [hunk 2]
```diff
@@ -104,8 +104,12 @@ function Home(): ReactElement {
                 >
                   Find answers to frequently asked questions
                 </Link>,
-                <Link href={sblHelpLink} key='ask-a-question' type='list'>
-                  Contact our support staff
+                <Link
+                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Error page: encountered an error on the SBL Platform'
```
Comment (meissadia) [inline src/pages/Filing/FilingHome.tsx:108]: Link is on Filing homepage:
```suggestion
                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Filing home page: Seeking guidance'
```

**Review by meissadia — CHANGES_REQUESTED** [review]
One email subject line feels out of alignment with it's location but, otherwise, still looking great.

File: src/pages/Filing/FilingHome.tsx (hunk 2 above)
Comment (billhimmelsbach) [reply src/pages/Filing/FilingHome.tsx:108]: Not sure how I missed that, good catch!

Comment (billhimmelsbach) [reply src/pages/Filing/FilingHome.tsx:108]: I'm also fan of "Seeking guidance" @meissadia 

Screenshot 2024-03-29 at 2 36 53 PM

**Review by meissadia — DISMISSED** [review]

File: src/pages/Filing/FilingHome.tsx [hunk 3]
```diff
@@ -105,7 +105,7 @@ function Home(): ReactElement {
                   Find answers to frequently asked questions
                 </Link>,
                 <Link
-                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Filing home page: Seeking guidance'
+                  href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Platform home: Get help'
```
Comment (meissadia) [inline src/pages/Filing/FilingHome.tsx:108]: 😢

**Pull request merged**